app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,24 @@
+.govuk-related-items {
```

Comment:
I've noticed our current margins are responsive, but the gutters are fixed pixels – updating to `$gutter` _feels_ like a regression.